src/scripts/index.js

```diff
@@ -0,0 +1,20 @@
+import createTag from "./utils/createTag.js";
+
+//Send request DOM API elements
+const divSend = createTag({ className: "div-send-request" });
+const spanSelectAndInput = createTag({
+  tagName: "span",
+  className: "span-select-and-input",
+});
+const selectSend = createTag({ tagName: "select", tagId: "select-send" });
```

Comment:
According to first comment. This is element of our component, so using BEM -> 'send__methods'

Reply:
Done